libs/model/src/models/notification.ts

```diff
@@ -36,8 +36,12 @@ export default (
     'Notification',
     {
       id: { type: dataTypes.INTEGER, primaryKey: true, autoIncrement: true },
-      notification_data: { type: dataTypes.TEXT, allowNull: false },
-      chain_event_id: { type: dataTypes.INTEGER, allowNull: true },
+      notification_data: { type: dataTypes.TEXT, allowNull: true },
+      chain_event_id: {
+        type: dataTypes.INTEGER,
+        allowNull: true,
+        unique: true,
```

Comment:
I think defining `unique: true` here is actually creating duplicate indexes... Additionally, as mentioned in another comment, defining indexes below provides more flexibility in terms of configuration than defining them here in the attributes.

Reply:
same issue with EvmEventStores, index or constaint? ...and might need migration according to answer

Reply:
Let's leave it like this for now though noting that we won't me able to match constraint names in the future.